Runtime/Scripts/CarrierInfoIntegration.cs

```diff
@@ -178,6 +178,10 @@ public int GetSignalStrength()
         return -1;
       }
       AndroidJavaObject signalStrength = telManager.Call<AndroidJavaObject>("getSignalStrength");
+      if (signalStrength == null)
```

Comment:
Shall you check for null or INVALID as mentioned in  Android Docs
https://developer.android.com/reference/android/telephony/SignalStrength
<img width="840" alt="Screen Shot 2021-08-24 at 10 33 17 AM" src="https://user-images.githubusercontent.com/24863504/130663074-d60e4249-4dda-44b6-912e-4b1396afbc0e.png">

Reply:
SignalStrength's documentedabstracted public getLevel() range is [0-4] inclusive. No SIM or TelManager --> -1. Next to no signal? 0. I guess we happily return INVALID as well depending on getLevel being documented correctly or not, so at least it's distinguishable in the backend. We'll need to see more platforms.